low-level/message_handlers/service_msg_handler.py

```diff
@@ -129,32 +146,41 @@ def _process_msg(self, jsonMsg):
         if jsonMsg.get("sspl_ll_msg_header") is not None and \
            jsonMsg.get("sspl_ll_msg_header").get("uuid") is not None:
             uuid = jsonMsg.get("sspl_ll_msg_header").get("uuid")
-            self._log_debug(f"_processMsg, uuid: {uuid}")
+            logger.debug(f"_processMsg, uuid: {uuid}")
 
         # Handle service start, stop, restart, status requests
         if jsonMsg.get("actuator_request_type").get("service_controller") is not None:
-            self._log_debug("_processMsg, msg_type: service_controller")
+            logger.debug("_processMsg, msg_type: service_controller")
 
             service_name = jsonMsg.get("actuator_request_type") \
                 .get("service_controller").get("service_name")
             service_request = jsonMsg.get("actuator_request_type") \
                 .get("service_controller").get("service_request")
             request = f"{service_request}:{service_name}"
 
+            if service_name not in self.monitored_services:
+                logger.error(f"{service_name} - service not monitored")
+                msg = "Check if supplied service name is valid, %s is not monitored or managed." % service_name
+                self.send_error_response(service_request, service_name, msg, errno.EINVAL)
+                return
+            elif service_request not in ["disable","enable"]:
+                status = Service('dbus').check_service_is_enabled(service_name)
+                if status == "disabled":
+                    logger.error(f"{service_name} - service is disabled")
+                    msg = "Service is disabled, first send the request to enable the service, and then to start the service."

```

Comment:
msg = "%s service is disabled, 'enable' request needed before current '%s' request can be processed" % (service_name, service_request )

Reply:
done.